ansible/environments/group_vars/all/vars.yml

```diff
@@ -6,6 +6,14 @@ PORT_NO: 8081
 #Set verbisity level and run playbooks with -vv option to display custom debug messages
 verbosity_level: 2
 
+#Details for Nova Compute filter
+#specify NOVA_COMPUTE_FILTERS_FILE
+#For Debian: /etc/nova/rootwrap.d/compute.filters
+#For RHEL : /usr/share/nova/rootwrap/compute.filters 
+NOVA_COMPUTE_FILTERS_FILE: /usr/share/nova/rootwrap/compute.filters
+FILTERS_FILE_Path_RHEL: /usr/share/nova/rootwrap/trilio.filters
+FILTERS_FILE_Path_Debian: /etc/nova/rootwrap.d/trilio.filters
```

Comment:
coding standard: keep the case uniform for variable names.
If we are opting upper case, then make sure all the alphabets are in upper case. Don't mix the cases. e.g: FILTERS_FILE_Path_Debian  should be FILTERS_FILE_PATH_DEBIAN.

Reply:
Suggested code changes done